app/controllers/ops_controller/settings/zones.rb

```diff
@@ -28,8 +28,10 @@ def zone_edit
         javascript_flash(:spinner_off => true)
         return
       end
-      # zone = @zone.id.blank? ? Zone.new : Zone.find(@zone.id)  # Get new or existing record
+
+      zone_save_ntp_server_settings(@zone)
```

Comment:
I think we need this after the `@zone.save` to avoid the case where the zone does not yet have an id, right?

Reply:
Aye, yeah, oops.